### PR TITLE
Funnel -> core

### PR DESCRIPTION
**Pull request by YoniKiriaty**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/funnel/FunnelCommands.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	}
+
+	public Command setPower(double power) {
+		return new FunctionalCommand(() -> {}, () -> funnel.setPower(power), (interrupted) -> funnel.stop(), () -> false, funnel);
+	}
+
+	public Command stop() {
+		return new InstantCommand(() -> funnel.stop(), funnel);
+	}
+
```
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/funnel/FunnelCommands.java]: space

File: src/main/java/frc/robot/subsystems/funnel/FunnelConstants.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package frc.robot.subsystems.funnel;
+
+
+public class FunnelConstants {
+
+	public static final String LOG_PATH = "Subsystems/funnel/";
+
+	public static final boolean ENABLE_FORWARD_LIMIT_SWITCH = false;
```
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/funnel/FunnelConstants.java]: u dont have limit

**Comment by ItayKauff** [comment]
Why arent you on CodeGuru✖️?

File: src/main/java/frc/robot/subsystems/funnel/FunnelCommands.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		this.funnel = funnel;
+	}
+
+	public Command setPower(double power) {
+		return new FunctionalCommand(() -> {}, () -> funnel.setPower(power), (interrupted) -> funnel.stop(), () -> false, funnel);
+	}
+
+	public Command stop() {
+		return new InstantCommand(() -> funnel.stop(), funnel);
+	}
```
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/funnel/FunnelCommands.java:21]: add with name

File: src/main/java/frc/robot/subsystems/funnel/FunnelCommands.java
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/funnel/FunnelCommands.java:1]: remove this lcass

**Pull request merged**